ui.frontend/pom.xml

```diff
@@ -146,6 +130,7 @@
                                     <arguments>
                                         <argument>link</argument>
                                         <argument>@adobe/aem-core-cif-react-components</argument>
+                                        <argument>@adobe/cif-product-recs-extension</argument>
```

Comment:
What is the reason that we don't name both packages the same way? 

`aem-core-cif-product-recs-extension`

Reply:
@dplaton wanted to have a shorter name: https://github.com/adobe/aem-core-cif-components/pull/605#discussion_r665994699

Reply:
I agree on that, but I think we should at least retain the prefix `aem-core-cif`

Reply:
In fact it looks like all aem related packages on npm have at least the `aem-` prefix https://www.npmjs.com/search?q=%40adobe%2Faem

Reply:
@Buuhuu Daniel made the point that the `aem-` prefix is not needed, since `cif` only works with AEM anyway. 

I'm fine with changing the name again, but please agree on a name with @dplaton first, so we can keep the changes to be done in code to a minimum.

Reply:
I stand by my comment. Typing `npm install @adobe/aem-core-cif-product-recs-extension` is quite some work...

Reply:
I doubt that typing that is a common use case. In most cases you will copy past it. 

I have no hard feelings about naming. If there is no naming policy in place, we should still make sure to name the modules we own the same way and not randomly based on a subjective opinion of "too long".

Reply:
Package name was changed to `@adobe/aem-core-cif-product-recs-extension`.